infra/client/product-planner-ng/src/app/app-routing.module.ts

```diff
@@ -16,6 +16,7 @@ import {ManufacturerResolver} from './items/manufacturer.resolver';
 import {ItemStockResolver} from './items/item-stock.resolver';
 import {ItemStockListComponent} from './items/item-stock-list/item-stock-list.component';
 import {ItemStockListResolver} from './items/item-stock-list.resolver';
+import {EditCategoriesComponent} from './categories/edit-categories/edit-categories.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/maciejmalecki/inventory/pullRequest?prid=7113829)
```suggestion
import {EditCategoriesComponent} from "./categories/edit-categories/edit-categories.component";
```